src/sparql/qc/mondo/qc-ordo-subset-exact-mapping.sparql

```diff
@@ -0,0 +1,37 @@
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix oboInOwl: <http://www.geneontology.org/formats/oboInOwl#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+## This QC check ensures that if we have a source for subset, it must 
+## also be mapped to the same term as the subset
+
+SELECT DISTINCT ?entity ?property ?value WHERE {
+    VALUES ?subset {
+        <http://purl.obolibrary.org/obo/mondo#ordo_disorder>
+    }
+    ?entity oboInOwl:inSubset ?subset .
+
+    ?subset_anno a owl:Axiom ;
+           owl:annotatedSource ?entity ;
+           owl:annotatedProperty oboInOwl:inSubset ;
+           owl:annotatedTarget ?subset ;
+           oboInOwl:source ?xref .
+
+    FILTER NOT EXISTS {
+        ?entity oboInOwl:hasDbXref ?xref .
+        VALUES ?mondo_source {
+            "MONDO:obsoleteEquivalent"
+            "MONDO:equivalentTo"
```

Comment:
For now, from @sabrinatoro we should leave this as `MONDO:equivalentTo` until the obsolete Mondo terms that are in an ordo subset ([issue 7693](https://github.com/monarch-initiative/mondo/issues/7693)) are reviewed.

After those are reviewed, this `MONDO:obsoleteEquivalent` should be added back into the query (TODO: Add ticket for this work and link as step to #7693).

Reply:
Actually I think the second line should have been `After those are reviewed, this `MONDO:obsoleteEquivalent` could be removed from the query since it's only needed for now while those obsolete terms are being reviewed.